src/experiment/procedures/honeycombProcedure.js

```diff
@@ -39,12 +38,13 @@ export const buildHoneycombProcedure = () => {
     choices: honeycombSettings.timeline_variables.map((variable) => variable.correct_response),
     data: {
       // Record the correct_response passed as a timeline variable
-      code: eventCodes.honeycomb,
+      code: honeycombSettings.code,
       correct_response: getJsPsych().timelineVariable("correct_response"),
     },
     on_load: function () {
       // Conditionally flashes the photodiode when the trial first loads
-      if (ENV.USE_PHOTODIODE) pdSpotEncode(eventCodes.honeycomb);
+      if (ENV.USE_PHOTODIODE) pdSpotEncode(honeycombSettings.code);
+      console.log("honeycomb procedure code: " + honeycombSettings.code);
     },
```

Comment:
Just removing a console log
```suggestion
      if (ENV.USE_PHOTODIODE) pdSpotEncode(honeycombSettings.code);
    },
```